capstone-backend/src/main/java/com/google/sps/servlets/MeetingServlet.java

```diff
@@ -138,5 +147,14 @@ public void doPut(HttpServletRequest request, HttpServletResponse response) thro
   @Override
   public void doDelete(HttpServletRequest request, HttpServletResponse response) throws IOException {
     Utility.delete(meetings, request, response);
+    JsonObject jsonObject = Utility.createRequestBodyJson(request);
```

Comment:
nit: `jsonObject` could probably be named after the data it describes, maybe something to do with "calendar"

Reply:
Good idea, I'll change this.